### PR TITLE
Setup backend, knex, typescript client, etc.

### PR DESCRIPTION
**Pull request by t-laird**
Please ignore any configuration .xml files.

This PR will not make use of NGINX, I'll continue configuring that later if needed.

I reinstalled the client portion of the project, now located in `client/`. You may find that the typescript CRA is a little abrasive to work with initially. Check the console for relevant errors or drop a message in our slack and I'm sure we can collectively figure out React typings :). 

I used a boilerplate I made previously with a few small adjustments. There's a little bit of explanation on that repo's README [here](https://github.com/t-laird/Express-CRA-Boilerplate).

To sum it up simply, `npm install` in the root and `client/` directories and then run `npm run dev`.

I think that's everything, feel free to leave comments or DM me if you have questions.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,20 @@
+MIT License
+
+Copyright (c) 2018 Thomas Laird
```
Comment (katiescruggs) [inline LICENSE]: All of us?

File: server.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.set('port', process.env.port || 5000);
+app.listen(app.get('port'));
+
+app.use('/', express.static(`${__dirname}/client/build`));
+
+app.get('/api/v1/test', (req, res) => {
+  return res.status(200).json({ status: 'success' });
```
Comment (katiescruggs) [inline server.js]: I don't think we need the `return` or `status(200)` (pretty sure express defaults to 200 status), but they're not hurting anything.

**Review by katiescruggs — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Biz-Luminate",
+  "version": "1.0.0",
+  "description": "",
+  "author": "Natalie Montoya, Katie Scruggs, Maria St. Louis-Sanchez, Jesse Packwood, Megan Arellano, Matt Andrews, Thomas Laird",
+  "main": "knexfile.js",
+  "scripts": {
+    "test": "mocha --exit",
+    "server": "nodemon server.js",
+    "client": "cd client && yarn start",
```
Comment (Biz-Luminate) [inline package.json]: I was getting an error for yarn start. I changed it locally to npm start to get npm run dev to work.
